packages/scratch-vm/src/extensions/scratch3_spotify/index.ts

```diff
@@ -0,0 +1,492 @@
+import { ArgumentType, BlockType } from "../../typescript-support/enums";
+import { Extension } from "../../typescript-support/Extension";
+import {
+    Block,
+    DefineBlock,
+    Environment,
+    ExtensionMenuDisplayDetails,
+} from "../../typescript-support/types";
+import defineTranslations from "./translations";
+import Tone from "./tone.js";
```

Comment:
We should be able to just `npm install tone` and then make use of it (you'll just perform install from the `packages/scratch-vm` location).

Like described here: https://github.com/Tonejs/Tone.js

Though, one thing I need to check is how to correctly import it, since it recommends doing:
```ts
import * as Tone from 'tone';
```
Which I believe will work fine! But sometimes how you configure typescript causes it to forbid `* as ___` imports and then you need to try an alternative. I can give this a shot next week! 

Though one think 

It looks like the lastest

Reply:
Confirmed 
```ts
import * as Tone from 'tone';
```
works!